src/test/java/core/basesyntax/dto/CustomWriterFileImpTest.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.dto;
+
+import core.basesyntax.imp.CustomFileReadImp;
+import core.basesyntax.imp.FruitParseDtoParseImp;
+import core.basesyntax.service.CustomFileReader;
+import core.basesyntax.service.FruitParse;
+import java.util.List;
+import org.junit.Test;
+
+public class CustomWriterFileImpTest {
```

Comment:
I would suggest that you add one more test that checks your fileWriter with an invalid path
} catch (IOException e) {
  throw new RuntimeException("Can't write file", e); test this
}